conbench/api/_examples.py

```diff
@@ -159,7 +159,7 @@ def _api_commit_entity(commit_id, parent_id, links=True):
             "self": "http://localhost/api/commits/%s/" % commit_id,
         },
         "branch": "some_user_or_org:some_branch",
-        "fork_point_sha": "some_fork_point_sha",
+        "fork_point_sha": "02addad336ba19a654f9c857ede546331be7b631",
```

Comment:
This PR is relying on the [assumption](https://github.com/conbench/conbench/pull/431#discussion_r1021908358) that a commit's sha == its (non-None) fork_point_sha if and only if it's on the default branch. I had to update our fake commits to reflect that.

Reply:
Quick note about terminology. The Internet often uses "sha" instead of "hash", whereas "hash" really is the better word.

I trust we all know this, but "sha" is actually SHA which is a rather well-known abbreviation for a very specific set of _algorithms_.  That is, "a sha" is not "a hash", because the hash is the output of the algorithm. Well well. :)

Reply:
That is, in code I kind of find `_hash` much better than `_sha`. I totally understand if that sounds like a huge nono annoyance .

Reply:
Haha, yes I thought the same thing when I onboarded onto this codebase. But "sha" is EVERYWHERE (not only in this repo, but also in many other repos that use Conbench)